ui/pages/home/home.component.js

```diff
@@ -814,15 +817,22 @@ export default class Home extends PureComponent {
     const tabPadding = process.env.MULTICHAIN ? 4 : 0; // TODO: Remove tabPadding and add paddingTop={4} to parent container Box of Tabs
 
     ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-    const showWhatsNew =
+    const canSeeModals =
       completedOnboarding &&
       (!onboardedInThisUISession ||
         firstTimeFlowType === FirstTimeFlowType.import) &&
-      announcementsToShow &&
-      showWhatsNewPopup &&
       !process.env.IN_TEST &&
       !newNetworkAddedConfigurationId;
 
+    const showSmartTransactionsOptInModal =
+      canSeeModals && isSmartTransactionsOptInModalAvailable;
+
+    const showWhatsNew =
+      canSeeModals &&
+      announcementsToShow &&
+      showWhatsNewPopup &&
+      !showSmartTransactionsOptInModal;
```

Comment:
Does this need to be shown on the home screen?  This will result in the user getting hit with multiple modals in a row.  Why can't this just be a regular "What's new" item?